### PR TITLE
Changed Pesto logotype's position.

### PR DESCRIPTION
**Pull request by dragostis**
The padding's role is to make sure that the logotype does not run
below the actual symbol.

@mpcomplete PTAL.

### CODE REVIEW

**Comment by mpcomplete** [comment]
IMO this makes the pesto logo look weird in its expanded state (compare to the reference impl https://material-adaptive.firebaseapp.com/pesto/app/#/home ). Can we instead play with the curves so that either:
a. the text fades out before touching the leaf icon, or
b. the icon shrinks fast enough that the text never reaches it?

**Comment by dragostis** [comment]
Variant *a* looks weird since I tried that first. I'll try to implement *b* and then push.

**Comment by dragostis** [comment]
@mpcomplete Done.

File: examples/flutter_gallery/lib/demo/pesto_demo.dart [hunk 1]
```diff
@@ -211,13 +214,13 @@ class _PestoLogoState extends State<PestoLogo> {
           overflow: Overflow.visible,
           children: <Widget>[
             new Positioned.fromRect(
-              rect: _imageRectTween.lerp(config.t),
+              rect: _imageRectTween.lerp(config.t.clamp(0.0, 1.0)),
```
Comment (mpcomplete) [inline examples/flutter_gallery/lib/demo/pesto_demo.dart]: IMO, it'd be better to clamp this in the PestoLogo constructor.

**Review by mpcomplete — COMMENTED** [review]
Looks much better! One small nit, then LGTM.

File: examples/flutter_gallery/lib/demo/pesto_demo.dart (hunk 1 above)
Comment (dragostis) [inline examples/flutter_gallery/lib/demo/pesto_demo.dart]: Fixed.

**Review by mpcomplete — APPROVED** [review]

**Pull request merged**